Packs/Trello/Classifiers/classifier-Trello_-_Incoming_Mapper.json

```diff
@@ -0,0 +1,65 @@
+{
+	"description": "",
+	"feed": false,
+	"id": "Trello - Incoming Mapper",
+	"mapping": {
+		"Trello": {
+			"dontMapEventToLabels": true,
+			"internalMapping": {
+				"Trello Board ID": {
+					"complex": null,
+					"simple": "create_action.board_id"
+				},
+				"Trello Board Name": {
+					"complex": null,
+					"simple": "create_action.list_name"
+				},
+				"Trello Card ID": {
+					"complex": null,
+					"simple": "id"
+				},
+				"Trello Card URL": {
+					"complex": null,
+					"simple": "url"
+				},
+				"Trello Created By": {
+					"complex": null,
+					"simple": "create_action.memberCreator.fullName"
+				},
+				"Trello Details": {
+					"complex": null,
+					"simple": "desc"
+				},
+				"Trello ID": {
```

Comment:
the Trello ID field exist in the mapper but it does not exist in the incident fields
should it be removed from the mapper or added to the PR?